core/src/main/resources/jenkins/management/ShutdownLink/index.groovy

```diff
@@ -0,0 +1,47 @@
+package jenkins.management.ShutdownNewLink
+
+import jenkins.management.Messages
+
+def f = namespace(lib.FormTagLib)
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+l.layout(norefresh: true, permission: app.ADMINISTER, title: my.displayName) {
```

Comment:
```suggestion
l.layout(norefresh: true, permission: app.MANAGE, title: my.displayName) {
```

Should be tested with Overall/Manage permission enabled (`Jenkins.MANAGE.enabled = true` + `matrix-auth` plugin for fine-grained permissions + a second user without Overall/Administer).

Reply:
- [ ] Update permission
Seems like it's working